app/__main__.py

```diff
@@ -51,6 +54,13 @@ def get_routes():
             AdminObjectPutHandler,
         ),
         (r"/api/admin/v1/scene/([0-9]{1,16})/duplicate", AdminSceneDuplicateHandler),
+        (
+            r"/static/(.*)",
+            UIStaticHandler,
+            dict(path=f"{os.path.dirname(__file__)}/public/static/"),
```

Comment:
Is this using our local path? When this is deployed on a server, what is this going to be?

Reply:
This is the path of the file where it this file exists (__main__.py). So I am saying get the directory where __main__.py exists, and append `public/static` to it. Meaning it will always be correct, regardless of running locally or on server